apps/src/templates/projects/NewProjectButtons.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import i18n from "@cdo/locale";
+import styleConstants from '../../styleConstants';
```

Comment:
This is somewhat tangential to this particular PR, but I've noticed sometimes we like to use `../` to navigate back to files like this, and sometimes we use `@cdo`. I believe you've said you prefer `../` because your editor handles it better (which is a legit argument). I've always had a preference for `@cdo` because it makes it super easy to copy paste from one file to another, and not need to worry about if the files are at the same depth in the tree.

Not sure if we actually want to settle on one or another of these, or just keep doing whichever.